source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -1,6 +1,9 @@
 /*
-Zividbenchmarks is a sample that will test the average speed of different operations on your computer. It will provide
-the mean and median for connects, disconnects, single imaging, HDR and filtering.
+ZividBenchmark is a sample that will test the performance of different operations on your computer. It will provide
```

Comment:
There are extra spaces and we can align the text. Also, we should use simple present tense, you can use this:

ZividBenchmark is a sample that tests the performance of different SDK operations on your computer. It provides measured time
in milliseconds for: connecting and disconnecting the camera, capture assistant, single and HDR captures with and without
filters, as well as saving point clouds.